core/src/main/java/org/elasticsearch/common/ParseField.java

```diff
@@ -29,6 +29,12 @@
  * Holds a field that can be found in a request while parsing and its different variants, which may be deprecated.
  */
 public class ParseField {
+    // todo reuse these commonly used ParseField objects instead of duplicating them in other classes
+    public static final ParseField FIELD_FIELD = new ParseField("field");
+    public static final ParseField FORMAT_FIELD = new ParseField("format");
+    public static final ParseField MISSING_FIELD = new ParseField("missing");
+    public static final ParseField TIME_ZONE_FIELD = new ParseField("time_zone");
+    public static final ParseField VALUE_TYPE_FIELD = new ParseField("value_type", "valueType");
```

Comment:
I set camelCase `valueType` here because it wasn't removed from the original parser: https://github.com/elastic/elasticsearch/blob/master/core/src/main/java/org/elasticsearch/search/aggregations/support/AbstractValuesSourceParser.java#L121

I thought it best to set as deprecated here but based on the feedback I presume its OK to simply remove it?